internal/ackhandler/packet.go

```diff
@@ -10,7 +10,7 @@ import (
 // A Packet is a packet
 type Packet struct {
 	PacketNumber    protocol.PacketNumber
-	Frames          []Frame
+	Frames          []*Frame
```

Comment:
It's a bit sad that we need to use a pointer here now, but that's the price we're paying for not allocating.